.github/workflows/dev-arm64.yml

```diff
@@ -0,0 +1,36 @@
+name: dev ARM64 build
```

Comment:
+1 on combining the workflows into a single one. As discussed off-line, there are some quick wins that could resolve the performance issues on cross-emulation.